src/features/genresSlice.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const genresSlice = createSlice({
+    name: "genres",
+    initialState: {
+        genres: [],
+        status: "initial",
+    },
+
+    reducers: {
+        fetchGenres: () => ({
+            status: "loading",
+        }),
+
+        fetchGenresSuccess: (_, { payload: genres }) => ({
+            status: "success",
+            genres,
+        }),
+
+        fetchGenresError: () => ({
+            status: "error",
+        }),
+    },
+});
+
+export const {fetchGenres, fetchGenresSuccess, fetchGenresError} = genresSlice.actions;
+
+export const selectGenres = (state) => state.genres.genres;
+export const selectGenresStatus = (state) => selectGenres(state).status;
```

Comment:
tu wykorzystujesz tą linijkę wyżej a skoro ona jest źle to tutaj też nie możesz dobić się do statusu